inst/htmlwidgets/leaflet.js

```diff
@@ -534,6 +558,7 @@ _htmlwidgets2.default.widget({
         }
 
         preventUnintendedZoomOnScroll(map);
+        setupShowHideGroupsOnZoom(map);
```

Comment:
One thing I am a bit concerned about is, that this sets up the `zoomend` event listener, no matter whether any groups were added with zoomLevels or not.
Can't this whole thing be setup from `methods.setGroupOptions`, so that the event listener is registered only when someone sets up groupOptions?

Other than that looks good.